protocol/packages/swap/Cargo.toml

```diff
@@ -18,47 +18,45 @@ combinations = [
 feature-groups = { testing = { members = ["testing"], at-least-one = false, mutually-exclusive = false } }
 
 [features]
-# TODO get rid of main and test and specialize astroport into ~_main and ~_test when the CI builds protocols provided as input
-# instead of implicitly building them in matrix
-main = []
-test = []
-
-astroport = [
+# Mandatory features
+dex-astroport_test = [
+    "dep:astroport",
     "sdk/cosmos_ibc",
+]
+dex-astroport_main = [
     "dep:astroport",
-    "dep:currency",
-    "dep:dex",
-    "dep:finance",
-    "dep:oracle",
-    "dep:platform",
-    "dep:sdk",
-    "dep:serde",
+    "sdk/cosmos_ibc",
 ]
```

Comment:
I am curious why the test dex would require `"sdk/cosmos_ibc"` and the main dex not.